spec/sandbox_5_2/Gemfile

```diff
@@ -3,7 +3,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "2.6.8"
+ruby "2.6.9"
```

Comment:
Good call! I wanted to keep it at a lower version for now since the required version for the gem is ruby 2.5 or greater (I didn't want to downgrade at the moment). Once I get everything setup with GitHub Actions to run specs on different ruby versions, then ya, I think it would be good to update this.